docs/reference/api/docker_remote_api_v1.18.md

```diff
@@ -1501,7 +1501,7 @@ Search for an image on [Docker Hub](https://hub.docker.com).
 
 **Example request**:
 
-    GET /images/search?term=sshd HTTP/1.1
+    GET /v1.18/images/search?term=sshd HTTP/1.1
```

Comment:
hmf, this is gonna be a nightmare to keep the API docs in sync. Well, until we move to Swagger :smile: